pkg/core/native/native_neo.go

```diff
@@ -425,17 +429,52 @@ func (n *NEO) OnPersist(ic *interop.Context) error {
 		cache := ic.DAO.GetRWCache(n.ID).(*NeoCache)
 		// Cached newEpoch* values always have proper value set (either by PostPersist
 		// during the last epoch block handling or by initialization code).
+
+		committeeChanged, prevCommitteeStackItem, newCommitteeStackItem := n.checkCommitteeChange(cache)
+
 		cache.nextValidators = cache.newEpochNextValidators
 		cache.committee = cache.newEpochCommittee
 		cache.committeeHash = cache.newEpochCommitteeHash
 		cache.votesChanged = false
 
+		if committeeChanged {
+			// Notify subscribers if the committee has changed.
+			ic.AddNotification(n.Hash, "CommitteeChanged", stackitem.NewArray([]stackitem.Item{
+				prevCommitteeStackItem, newCommitteeStackItem,
+			}))
+		}
 		// We need to put in storage anyway, as it affects dumps
 		ic.DAO.PutStorageItem(n.ID, prefixCommittee, cache.committee.Bytes(ic.DAO.GetItemCtx()))
 	}
 	return nil
 }
 
+// checkCommitteeChange checks if the committee has changed and returns the previous and new committee as stackitem.Item.
+func (n *NEO) checkCommitteeChange(cache *NeoCache) (bool, stackitem.Item, stackitem.Item) {
```

Comment:
Don't need a separate function, because no one else uses it and it's small. Besides, its result strictly depends on committee cache update and may be wrong if called in wrong place.